### ISSUE TITLE
Cray XT configure flag apply to XE/XK/XC?

### ISSUE THREAD

**Issue by jeffhammond** [LLNL/STAT#5]
Is flag truly specific to Cray XT or is it the flag for Cray X[TEKC] supercomputers in general?
```
--with-cray-xt          Add the flags to run on a Cray XT system
```
From the source, it would appear to be the flag to use with the Cray ALPS scheduler, which was used on Cray X[TEKC] systems, but has been phased out in favor of SLURM, at least on NERSC Cray XC systems.

If it is a flag for Cray ALPS, it would be helpful to users to rename it to `--with-cray-alps` to ensure that it is:
* used on Cray X[EKC] systems that use ALPS.
* not used on Cray systems that use SLURM instead of ALPS.

Thanks,

Jeff

**Comment by lee218llnl** [comment]
@jeffhammond Good point, I will updated this in the next release. Thanks.

**Comment by lee218llnl** [comment]
I renamed this to --with-cray-alps in https://github.com/LLNL/STAT/commit/8b059441ad261206dd80cb8d26347cef54e74ce9.

**Comment by dongahn** [comment]
@lee218llnl: I think it would be ideal if can make this flag go away since LaunchMON should be able to detect the underlying RM at runtime on a given platform. What are the STAT features enabled when this config flag is given?

**Comment by lee218llnl** [comment]
@dongahn This commit (https://github.com/LLNL/STAT/commit/52d57e0f6fa2473dedb3c63bcbcdeba967701880) replaces the static RM setting with runtime detection. I have tested this on LLNL Linux clusters and BG/Q systems and there is no regression there. After MUCH toil, I was able to get this commit of STAT and all its deps built on LANL's trinitite Cray system (using alps) and got it to run OK there too.
